FlowCal/mef.py

```diff
@@ -654,6 +737,9 @@ def get_transform_fxn(data_beads,
             FlowCal.plot.scatter3d_and_projections(
                 populations,
                 channels=clustering_channels[:3],
+                xscale='log' if clustering_amp_type[0][0] else 'logicle',
+                yscale='log' if clustering_amp_type[1][0] else 'logicle',
+                zscale='log' if clustering_amp_type[2][0] else 'logicle',
```

Comment:
same as above: i don't remember when `clustering_amp_type[0][0]` evaluates to `True`